packages/clay-css/src/scss/atlas/variables/_alerts.scss

```diff
@@ -27,7 +27,7 @@ $alert-close: map-deep-merge(
 	$alert-close
 );
 
-$alert-font-size: 0.875rem !default; // 14px
+$alert-font-size: 1rem !default; // 14px
```

Comment:
@pat270 is this expected? I noticed that the alerts will undergo a visual change, it seems to have gotten bigger, in addition to the font size that has also changed, see this comparison:

|. Before | After |
|--------|-------|
| <img width="990" alt="Screen Shot 2021-10-05 at 18 51 37" src="https://user-images.githubusercontent.com/13750819/136107865-cb14d26b-7685-4cb1-af8b-6725d7826c85.png"> | <img width="999" alt="Screen Shot 2021-10-05 at 18 52 24" src="https://user-images.githubusercontent.com/13750819/136107950-9a43db51-23f8-4daa-ad0d-bbfa6574eec0.png"> |

Reply:
Oh ok, no problem.

Reply:
This one should be good to go. I changed the wrong variable, should have been `$alert-indicator-font-size`. I also changed the padding so alert height should be the same.